design/restful-interface

```diff
@@ -0,0 +1,195 @@
+# Vela RESTful API design
+
+Below, I will list the Vela backend RESTful API grouped by each resource.
+
+We will set RESTful version as `APIVersionV1 = "2020-08-15"`
```

Comment:
I suggest use `v1` instead of `2020-08-15`

Reply:
This is the standard RESTful version practice for the version to be a date. I can move it our of the URL though.

Reply:
I never seen version in url path, the examples you give put it in header or parameters not url path. Let move it to header or parameters.